samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -1132,7 +1110,58 @@ static void anomaly_172_strict_mode_set(bool enable)
 
 	dtm_inst.strict_mode = enable;
 }
-#endif
+
+static void errata_172_handle(bool enable)
+{
+	if (!nrf52_errata_172()) {
+		return;
+	}
+
+	if (enable) {
+		if ((*(volatile uint32_t *)0x40001788) == 0) {
+			dtm_inst.anomaly_172_wa_enabled = true;
+		}
+	} else {
+		anomaly_172_strict_mode_set(false);
+		nrfx_timer_disable(&dtm_inst.anomaly_timer);
+		dtm_inst.anomaly_172_wa_enabled = false;
+	}
+}
+#else
+static void errata_172_handle(bool enable)
+{
+	ARG_UNUSED(enable);
+}
```

Comment:
Is it necessary to create an empty function?
In the `errata_172_handle` function there is a condition `if (!nrf52_errata_172())`, which seems to be sufficient.

Reply:
It is necessary, because there are references to anomaly_timer, which is not present in for example nRF5340 compiled code. This would cause build errors and warnings.